src/content/docs/features/state.md

```diff
@@ -2,7 +2,9 @@
 title: Durable State
 ---
 
-One of the fundamental features that separates Membrane from other serverless runtimes is that it's _stateful_. That means you can store state in one call to a [program](/concepts/programs/) and use it again in a subsequent call.
+One fundamental feature that separates Membrane from other serverless runtimes is that it is _stateful_.
+
+Programs have a `state` object that persists state between updates (i.e. deploying your program on save) and invocations (i.e. running your program manually or on a timer). To keep data around, put it in the `state` object, and that's it.
```

Comment:
I think it's important to clarify early on that the entire heap is durable. Not just `state`. Just so that users don't think that a long running task (i.e. `await sleep(ONE_DAY)`) needs to persist some data in `state` to be resilient. It's a common misconception.

I would even put this first, and talk about `state` after.